fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -236,32 +207,18 @@ private Template getTemplate(final Model rdf, final Node subject,
                                   .findFirst();
 
         if (!template.isPresent()) {
-            LOGGER.trace("Attempting to discover mixin types of node for resource in question: {}", subject);
-            template = rdf.listObjectsOfProperty(createResource(subject.getURI()),
-                                                 createProperty(getRDFNamespaceForJcrNamespace(JCR_NAMESPACE) +
-                                                     "mixinTypes"))
-                          .toList().stream()
-                          .map(x -> x.asLiteral().getLexicalForm())
+            LOGGER.trace("Attempting to discover RDF types of node for resource in question: {}", subject);
+            template = rdf.listObjectsOfProperty(subjectURI, type).toList().stream()
```

Comment:
That's more annoying. I wouldn't use Guava. I would use Jena's fluent `Iter`, e.g. `static import Iter.iter` and then
```
iter(listObjectsOfProperty(subjectURI, type))
    .map(Node::.getURI)
    .filter(typeMap::containsKey)
    .map(typeMap::get)
```
etc.